ci/praktika/result.py

```diff
@@ -424,7 +424,10 @@ def from_gtest_run(cls, unit_tests_path, name="", with_log=False):
             name=name,
             command=[
                 f"chmod +x {unit_tests_path}",
-                f"{unit_tests_path} --gtest_output='json:{ResultTranslator.GTEST_RESULT_FILE}'",
+                # With gdb we will capture stacktrace in case of abnormal termination and timeout (30 mins)
+                # 'catch syscall exit_group' is a hack to avoid exiting with non-zero exit code when process terminated successfully
+                # FIXME: do not run for sanitizers
```

Comment:
@maxknv I need some help with CI magic, can you please help me resolve this todo?

Reply:
Thanks!